web/cobrands/fixmystreet/assets.js

```diff
@@ -1344,7 +1344,7 @@ fixmystreet.message_controller = (function() {
         },
 
         asset_not_found: function() {
-            if (!this.visibility) {
+            if (this.fixmystreet.ignore_missing_asset || !this.visibility) {
```

Comment:
I don't think we need this variable - we shouldn't be calling asset_not_found at all if we don't care about it not being found :) Proposed solution elsewhere.